src/js/chrome.js

```diff
@@ -2,10 +2,15 @@ import { spinUpStore } from './redux-config';
 import * as actionTypes from './redux/action-types';
 import loadInventory from './inventory';
 import auth from './auth';
+import analytics from './analytics';
 import jwt from 'jwt-redhat';
 
 const onAuth = auth();
 
+onAuth.then(() => {
+    analytics(jwt.getUserInfo());
```

Comment:
Can you call the `getName` function from here? So the data will be loaded emediately when user logs in.